adapters/ownadx/ownadx.go

```diff
@@ -0,0 +1,165 @@
+package ownadx
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/prebid/openrtb/v19/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/macros"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"net/http"
+	"text/template"
+)
+
+type ownAdxAdapter struct {
```

Comment:
Scope of `ownAdxAdapter` is limited to this adapter package. Therefore `ownAdxAdapter` can be renamed to `adapter`. 

  ```
        type adapter struct {
          endpoint string
        }
       
 ```

Reply:
Rename ownAdxAdapter to adapter